README-Docker.md

```diff
@@ -0,0 +1,68 @@
+# Docker
+
+
+## Create Docker image
+
+* Assuming you have [Docker](https://docs.docker.com/get-docker/) installed locally.
+* First create an image tagged `fish`
```

Comment:
How about `fsh` instead of `fish` in order to make the image name correspond to FSH?